ember.md

```diff
@@ -98,6 +100,40 @@ get(this, 'isDestructured'); //true
 set(someObject, 'isUpdated', true);
 get(someObject, 'isUpdated'); //true
 ```
+
+<a name="data"></a>
+## Data
+
+<a name="data--data-binding"></a>
+### 3.1 Data Binding
+
+Any new code written should only contain one-way data-binding
+
+> Why? Two-way data-binding can have unexpected side effects. It is harder to debug, timing can be an issue on what value gets bound, and it can unknowingly create endless loops in your code.
```

Comment:
Might want to expand on the timing one - a specific case is if you pass the same data to 2 components and they both are allowed to change it. DDAU allows the parent to arbitrate those changes and resolve conflicts. Otherwise component A can, by the one-way binding chain, actually make changes in component B.

Reply:
Add race condition, cancel button example (maria --> chicken), remove loops or expand?

Reply:
can you also expand and include not using the `mut` helper?